uspv/msghandler.go

```diff
@@ -13,7 +13,10 @@ func (s *SPVCon) incomingMessageHandler() {
 		n, xm, _, err := wire.ReadMessageWithEncodingN(s.con, s.localVersion,
 			wire.BitcoinNet(s.Param.NetMagicBytes), wire.LatestEncoding)
 		if err != nil {
-			log.Printf("ReadMessageWithEncodingN error.  Disconnecting: %s\n", err.Error())
+			log.Printf("ReadMessageWithEncodingN error.  Disconnecting from given peer. %s\n", err.Error())
```

Comment:
this happens even if the peer isn't given, so maybe just say "disconnecting"
Except it may not even disconnect properly, so maybe we can add s.con.Close() too

Reply:
oh, ok, I left it because it would eventually fall through and disconnect, but makes sense to disconnect here as well